### PR TITLE
Add sane-doc-report backwards exception and update sane-doc-reports docker tag.

### PR DESCRIPTION
**Pull request by agamm**
## Status
Ready (after CR)

## Related Issues
https://github.com/demisto/etc/issues/19284
demisto/dockerfiles#476

## Description
Adds an exception for `sane-doc-reports` as it is an internal automation that requires docker tag changes quite often.

## Does it break backward compatibility?
   - Yes
       - Further details: Only for sane-doc-reports automation builds.

## Must have
- [X] Tests
- [ ] Code Review

## Additional changes
Updated `sane-doc-reports` automation to the newest docker tag.

## Docs:
[Scripts/SaneDocReport/CHANGELOG.md
](https://github.com/demisto/content/pull/4467/files#diff-7da22a34d43864471245196e7c4639ce)

### CODE REVIEW

File: Tests/scripts/hook_validations/tests/script_test.py [hunk 1]
```diff
@@ -49,6 +49,46 @@ def test_added_docker_image_on_existing_script():
...
+    validator.current_script = {
+        "type": "python",
+        "subtype": "python3"
+    }
+    validator.old_script = {
+        "type": "python",
+        "subtype": "python2"
+    }
+
+    assert validator.is_changed_subtype() is True, \
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/tests/script_test.py]: add another assert for `is_backward_compatible` return value (should be false)

File: Scripts/SaneDocReport/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Fixes table bugs.
```
Comment (yaakovi) [inline Scripts/SaneDocReport/CHANGELOG.md]:
```suggestion
Fixes table bugs.
```

File: Tests/scripts/hook_validations/tests/script_test.py (hunk 1 above)
Comment (agamm) [reply Tests/scripts/hook_validations/tests/script_test.py]: @yaakovi looks good?

**Comment by meirwah** [comment]
@agamm what is status here?

**Pull request merged**